code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -32,6 +32,12 @@ LINEN BINS
 	add_fingerprint(user)
 	return
 
+/obj/item/weapon/bedsheet/attackby(obj/item/I, mob/user)
+	if(istype(I, /obj/item/weapon/wirecutters) || istype(I, /obj/item/weapon/shard))

```

Comment:
honestly this is just placeholder until remie's PR is done so we can cut all this snowflake checking out and actually use the updated sharp stuff.